R/dataframe.R

```diff
@@ -229,6 +229,12 @@ as_tibble.default <- function(x, ...) {
 
 #' @export
 #' @rdname as_tibble
+as.tibble <- function(x, ...) {
```

Comment:
My way of adding `as.tibble` as an alias is different from `as_data_frame`, which is

```
as_data_frame <- function(x, ...) {
  UseMethod("as_data_frame")
}

#' @export
as_data_frame.tbl_df <- as_tibble.tbl_df
```
Is there any reason to go this more complicated route?